### PR TITLE
Moves HTML out of Main to Page.View module.

### PR DESCRIPTION
**Pull request by Shrikant9**
Issue [#475](https://github.com/bigbinary/acehelp/issues/475)

### CODE REVIEW

**Comment by Shrikant9** [comment]
@tejaswinichile Rebasing latest `master` branch changes into my branch solved the problem. Working fine on my local.

**Comment by Shrikant9** [comment]
@breezyboa _a

File: app/javascript/Admin/Main.elm [hunk 1]
```diff
@@ -177,6 +176,16 @@ setRoute location model =
         navigateTo newRoute model
 
 
+navigateOnClick : Route.Route -> Msg
+navigateOnClick route =
+    NavigateTo route
```
Comment (breezyboa) [inline app/javascript/Admin/Main.elm]: This function is not doing anything. We can directly use`NavigateTo`. Please remove the function

File: app/javascript/Admin/Main.elm [hunk 2]
```diff
@@ -177,6 +176,16 @@ setRoute location model =
...
+
+
+signOutClick : Msg
+signOutClick =
+    SignOut
```
Comment (breezyboa) [inline app/javascript/Admin/Main.elm]: Same goes for this function. We can directly use SignOut

**Comment by Shrikant9** [comment]
@breezyboa _a

**Pull request merged**